libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/LanguageGeneratorExtensions.cs

```diff
@@ -21,10 +21,12 @@ public static class LanguageGeneratorExtensions
         /// </summary>
         /// <param name="dialogManager">The <see cref="BotAdapter"/> to add services to.</param>
         /// <param name="defaultLg">Default LG Resource Id (default: main.lg).</param>
+        /// <param name="lazyMode">Should load the file in lazy mode.</param>
         /// <returns>The BotAdapter.</returns>
         public static DialogManager UseLanguageGeneration(
             this DialogManager dialogManager,
-            string defaultLg = null)
+            string defaultLg = null,
+            bool lazyMode = false)
```

Comment:
@Danieladu this code seems to be only used in tests and it is not clear to my why the tests fail if I try with lazyMode == true.

Lazy should be a performance optimization and it should work in both cases, lazy == false should load resources on demand and lazy == true should load them upfront, regardless, it should work in both cases unless we are doing something wrong.

Reply:
This parameter has been removed.